specificatie/verblijfplaats.yaml

```diff
@@ -129,54 +144,32 @@ components:
           properties:
             functieAdres:
               $ref: 'common.yaml#/components/schemas/Waardetabel'
-            adresregel1:
-              $ref: '#/components/schemas/Adresregel1'
-            adresregel2:
-              $ref: '#/components/schemas/Adresregel2'
-            woonplaats:
-              $ref: '#/components/schemas/Woonplaats'
     VerblijfplaatsBuitenlandBeperkt:
```

Comment:
NSwag genereert lege sub-types niet, de OpenAPI generator wel. Bij NSwag zie ik wel dat er een issue hiervoor is ingeschoten. Je kan erom heen werken door de `- type: object` regel te laten staan. De rest kan dan wel weg

Reply:
Voorstel doorgevoerd.